spec/features/user_can_update_posts_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.feature "Post Managment", type: :feature do
+  scenario "Can update posts" do
+    sign_up('test@example.com', 'password')
+    create_post('Hello, world!')
+    first(".post").click_on "Update"
+    fill_in "Message", with: "This is a different message"
+    click_button "Submit"
+    expect(page).to have_content "This is a different message"
+    expect(page).not_to have_content "Hello, world!"
+  end
+
+  scenario "Cannot update other users posts" do 
+    User.create(id: 1, name: 'Test Name', email: 'signin_test@example.com', password: '123456')
```

Comment:
Why not sign up (as in how come we're creating users directly into the database)?